README.md

```diff
@@ -1,38 +1,84 @@
-![Quire logo](static/img/quire_logo_tm.png)
+![](static/img/quire-logo--sm.png)
 
 # Quire
 
+**Create scholarly, visually rich digital books that last**
+
+Online • Print • E-book
+
 https://gettypubs.github.io/quire/
 
-**Quire is currently in closed development. To request access, please fill out our [Quire Beta Request](https://docs.google.com/forms/d/e/1FAIpQLSckvPWWyyfZJko6JTqf3slcXCV8vcCgQjAzoW4MfHEt9hDuxQ/viewform?usp=sf_link) form.**
+**Quire is currently in closed development. To request access, please fill out our [Quire Beta Request](https://docs.google.com/forms/d/e/1FAIpQLSckvPWWyyfZJko6JTqf3slcXCV8vcCgQjAzoW4MfHEt9hDuxQ/viewform) form.**
```

Comment:
Should this be "to get access" and not request?

Reply:
Good catch. I'll make this change and perhaps also look for other instances where this might make sense to change "request access" to "get access"